app/views/videos/show.html.slim

```diff
@@ -19,7 +19,7 @@
 
     .row
       .video-description.col-md-6.col-md-offset-3
-        = markdown video.description
+        p = simple_format(video.description)
```

Comment:
Before:

![Screenshot_2021-11-06_22-38-54](https://user-images.githubusercontent.com/1866809/140624761-88d08d4d-28c0-47aa-ae62-8ca62c6cc597.png)


After:

![Screenshot_2021-11-06_22-40-04](https://user-images.githubusercontent.com/1866809/140624762-bf255cb3-eba4-4284-a3db-fecc06f120d8.png)


Here there is a breaking change because markdown isn't supported anymore for video descriptions. This isn't a big deal only one video has it and it is old:

![Screenshot_2021-11-06_22-36-23](https://user-images.githubusercontent.com/1866809/140624779-698f29f6-ba73-4133-aa40-e0ebad801722.png)

https://rubyparis.org/videos/la-chabadabada-week-avec-simplon-co-vendredi-7-mars-de-18h-a-23h-samedi-8-mars-de-10h-a-17h